src/wl_window.c

```diff
@@ -2360,6 +2360,11 @@ void _glfwFocusWindowWayland(_GLFWwindow* window)
                     "Wayland: The platform does not support setting the input focus");
 }
 
+void _glfwDragWindowWayland(_GLFWwindow* window)
+{
+    xdg_toplevel_move(window->wl.xdg.toplevel, _glfw.wl.seat, _glfw.wl.pointerEnterSerial);
```

Comment:
Needs `_glfw.wl.serial` instead of `_glfw.wl.pointerEnterSerial`, else it does nothing.

Reply:
Added the suggestion